src/user.rs

```diff
@@ -536,7 +583,13 @@ fn pick_random_action(
 ) -> SocialAction {
     let mut rng = rand::thread_rng();
     if rng.gen_ratio(probability_to_act as u32, 100) {
-        if rng.gen_ratio(1, 75) {
+        if rng.gen_ratio(1, 3) {
+            SocialAction::SendMessage(RoomType::DirectMessage)
+        } else if channels_enabled && rng.gen_ratio(1, 5) {
+            SocialAction::SendMessage(RoomType::Channel)
+        } else if rng.gen_ratio(1, 3) {
+            SocialAction::AddFriend
+        } else if rng.gen_ratio(1, 75) {
```

Comment:
Is this change made on purpose? It might change the behavior significantly. Maybe it would be better to take to another PR.

Reply:
Okay, I will move this to another PR. You r right.